diff -urN work.orig/enc_jpeg2000.c work/enc_jpeg2000.c
--- work.orig/enc_jpeg2000.c	2018-07-08 07:19:02.714913907 -0400
+++ work/enc_jpeg2000.c	2018-07-08 07:20:38.997347460 -0400
@@ -121,7 +121,8 @@
     image.clrspc_=JAS_CLRSPC_SGRAY;         /* grayscale Image */
     image.cmprof_=0; 
 #endif
-    image.inmem_=1;
+/* See: http://www.zygrib.org/forum/viewtopic.php?f=3&t=1066 */
+/*    image.inmem_=1;  */
 
     cmpt.tlx_=0;
     cmpt.tly_=0;
